.github/workflows/release-to-sonatype.yml

```diff
@@ -0,0 +1,27 @@
+name: release-to-sonatype
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
+    inputs:
+      releaseversion:
+        description: 'Release version'
+        required: true
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Maven Central Repository
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
```

Comment:
in .github/workflows/maven.yaml we specify java-version 17
should we use 17 here too?

Reply:
my preference is that we keep with java 11 compliance for now.. let's create another issue to track updating to a newer version of java.

Reply:
Created LC-442 to track the task of evaluating Lucille for Java 17 upgrade.